src/main/resources/cru.properties

```diff
@@ -0,0 +1,12 @@
+# Theme name
```

Comment:
Correct. It's set as `cas.themeResolver.defaultThemeName=cru` in `cas.properties` (on the server).